hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1071,7 +1119,8 @@ public FileStatus getFileStatus(final Path path,
               blockSize,
               DateTimeUtils.parseLastModifiedTime(lastModified),
               path,
-              eTag);
+              eTag,
+              encryptionContext);
     }
```

Comment:
Inline with https://github.com/apache/hadoop/pull/3440#discussion_r989075168. Need to have encryptionContext in VersionedFileStatus object.